src/input-schemas.json

```diff
@@ -0,0 +1,820 @@
+{
+  "asset": {
+    "asset": {
+      "description": "Unique identifier with the name of the asset.",
+      "type": "VARCHAR"
+    },
+    "capacity": {
+      "constraints": {
+        "minimum": 0
+      },
+      "default": 0,
+      "description": "Capacity for one unit of the asset (is therefore multiplied by number of existing plus number of invested assets).",
+      "type": "DOUBLE",
+      "unit of measure": "MW"
```

Comment:
Keys should not have spaces. Use `unit_of_measure`. Otherwise everything looks good.